src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,13 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
```

Comment:
comment about semantic tags is not resolved
everything else looks good, so I will approve this PR

But take it into account in the future (semantic is important for SEO, accessibility)